src/cnxml/se.tsx

```diff
@@ -506,10 +506,19 @@ function rule(node: Rule, attrs: CommonAttrs, children: JSX.Node): JSX.Node {
 }
 
 function term(node: Term, attrs: CommonAttrs, children: JSX.Node): JSX.Node {
+    const nameIndexAttributes: { cxlxtName?: string, cxlxtBorn?: number, cxlxtDied?: number } = {}
+
+    if (node.index === 'name') {
+        nameIndexAttributes.cxlxtName = node.name
+        nameIndexAttributes.cxlxtBorn = node.born
+        nameIndexAttributes.cxlxtDied = node.died
+    }
+
     return <term
         xmlns={JSX.CNXML_NAMESPACE}
         cxlxtIndex={node.index}
         cmlnleReference={node.reference}
+        {...nameIndexAttributes}
         {...attrs}>
         { children }
```

Comment:
While we only allow these attributes on terms whose `index === 'name'`, it is not enforced by the XML schema, and I'm not sure whether it's worth adding extra complexity to the exporter to omit them if `index` is different.

Reply:
Since now they are split into separate interfaces I've had to either use `if` statement or cast the types to assigned these attributes

Reply:
Ah, since `Term extends Element`, `Term.name: unknown`, which implies `(Term | TermForNameIndex).name: unknown | undefined | string`, which is the same as `undefined`. 

Right, in this case that if may be the best solution. I'll just ask that you change type of `nameIndexAttributes` to `JSX.CXLXT.Attributes.NameIndex` (and perhaps rename it to something less verbose).

Reply:
done